pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -189,6 +189,9 @@ title: "Professional Development Workshop Survey"
           name: 'how_motivating_s',
           label: 'During your workshop, how motivating were the activities that this program had you do?'
 
+      - facilitators.each do |name|
+        %input{name: 'facilitator_names_s[]', type: 'hidden', value: name}
```

Comment:
Will this work, having multiple inputs of the same name? Should it be one input with an array of values?